### PR TITLE
Kg   shodenna 

### PR DESCRIPTION
**Pull request by Keeppie**
Dytini text and accordions are fully done!!!

Added Text and Accordions to
Diti
Igry
Pidlitki

New pages with buttons 
Diti
Pidlitki

Delited Antystres page as we don't need it in Dytini

Fixed some lines in Dytini

Multfilmi needs some imbedded multfilmi =)

### CODE REVIEW

**Review by Cu7ious — CHANGES_REQUESTED** [review]
Left comments on issue in our telegram dev channel

File: screens/nadzvychaynaSytuaziya/diyi/diyi.js [hunk 1]
```diff
@@ -23,17 +23,19 @@ export default function Diyi({ navigation }) {
...
 
-            <ListItem
-              roundTop
-              title="Вимкнули телебачення"
-              onPress={()=>navigation.push("Telebachennya")}
-            />
-            <ListItem title="Повітряна тривога" onPress={()=>navigation.push("Povitryana")} />
-            <ListItem title="Евакуація" onPress={navigateAway} />
-            <ListItem title="Вибухонебезпечні знахідки" onPress={()=>navigation.push("VibuhonebezpechniZnahidki")}/>
+            <text style={styles.paragraph}> Нас не готували до війни/кризи. Але ми вміємо просити та приймати
```
Comment (garys-primary) [inline screens/nadzvychaynaSytuaziya/diyi/diyi.js]: The component names are always capital, please rename `text` to be `Text` instead

**Review by garys-primary — DISMISSED** [review]
Please see comment:)

**Review by Cu7ious — APPROVED** [review]
LGTM

**Pull request merged**